setup.py

```diff
@@ -1,18 +1,19 @@
 #!/usr/bin/env python3
+from subprocess import call
+import locale
+import sys
 
+from setuptools import find_packages, setup
+from setuptools.command.test import test as TestCommand
+import setuptools.command.build_py
+
+from bears import assert_supported_version
+from bears import Constants
```

Comment:
It's dangerous because the imports may fail(because of version incompatibility) or any other reason ?